### DISCUSSION TITLE
Adding assets

### DISCUSSION THREAD

**Discussion by Tareksemaoui** [lowag/libremaint#9] [Q&A]
Hello , all the cmms work perfect , but  the asset page is empty and i can't find assets or   create new one , pls i need your help  thank you in advance .

**Comment by lowag** [comment]
Hello,
There should be a link on the left side "assets" and when you click on it you should see a "create new assets" just under the "assets". Is it not there?
Lowag

Androidos Outlookból<https://aka.ms/AAb9ysg> küldve
________________________________
From: Tareksemaoui ***@***.***>
Sent: Tuesday, June 20, 2023 7:30:53 PM
To: lowag/libremaint ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [lowag/libremaint] Adding assets (Discussion #9)


Hello , all the cmms work perfect , but the asset page is empty and i can't find assets or create new one , pls i need your help thank you in advance .

**Reply by Tareksemaoui** [reply]
there is  a link on the left side  'assets' and 'create new assets' but, the page is empty there is nothing , same problem on Notification and products.

**Reply by lowag** [reply]
Please check your webserver error log. If you are using linux you can see it with "tail /var/log/apache2/error.log". If it is complaining about permissions use this command (on linux systems): "chown -R www-data /var/www/libremaint"
